src/renderer/helpers/api/index.js

```diff
@@ -0,0 +1,8 @@
+import * as invidious from './invidious'
+import * as local from './local'
+
+export const channelShorts = async (api, channelId, idType, sortBy) => {
+  const shortsFunc = api === 'invidious' ? invidious.channelShortsInvidious : local.channelShortsLocal
+  const result = await shortsFunc(channelId, idType, sortBy)
+  return api === 'invidious' ? invidious.parseShortsResponse(result) : local.parseShortsResponse(result)
+}
```

Comment:
I'm also not a fan of this change, as it will have to be reverted when #3143 is merged as YouTube.js works differently to yt-channel-info.